src/client/client4.js

```diff
@@ -416,6 +416,13 @@ export default class Client4 {
         );
     }
 
+    updateServiceTermsStatus = async (userId, serviceTermsId, accepted) => {
```

Comment:
Shouldn’t this also be for the current user?

Reply:
Updated.